src/scenes/Partners/Detail/Tabs/People/PartnerDetailPeople.tsx

```diff
@@ -0,0 +1,84 @@
+import { Add } from '@mui/icons-material';
+import {
+  Button,
+  CardActionArea,
+  CardContent,
+  Skeleton,
+  Typography,
+} from '@mui/material';
+import { Many } from 'lodash';
+import { makeStyles } from 'tss-react/mui';
+import { square } from '~/common';
+import { Avatar } from '~/components/Avatar';
+import { UserListItemCardPortrait } from '~/components/UserListItemCard';
+import { EditablePartnerField } from '../../../Edit';
+import { PartnerDetailsFragment } from '../../PartnerDetail.graphql';
+
+interface Props {
+  partner: PartnerDetailsFragment | undefined;
+  editPartner: (item: Many<EditablePartnerField>) => void;
+}
+
+const useStyles = makeStyles()(({ spacing, palette }) => ({
+  sectionTitle: {
+    display: 'flex',
+    alignItems: 'center',
+    marginBottom: spacing(1),
+    '& > *': {
+      marginRight: spacing(2),
+    },
+  },
+  pocCardActionArea: {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  pocCardAvatar: {
+    ...square(86),
+    fontSize: 70,
+    color: palette.background.paper,
+  },
+}));
+
+export const PartnerDetailPeople = ({ partner, editPartner }: Props) => {
+  const { classes } = useStyles();
+
+  return (
+    <>
```

Comment:
The shadow on for the `UserListItemCardPortrait` is being cut off because the lack of space/padding under it. I recommend adding some bottom padding on the parent.
```suggestion
    <div css={(theme) => ({ paddingBottom: theme.spacing(2) })}>
```